templates/decisions/commentView.html.twig

```diff
@@ -1,49 +1,86 @@
 {% extends "base.html.twig" %}
 
-{% block title %}Decision
+{% block title %}Avis
 {% endblock %}
 {% block stylesheets %}
 	{{ parent() }}
-	{{ encore_entry_link_tags('decisionView') }}
+	{{ encore_entry_link_tags('commentView') }}
 {% endblock %}
-{% block body %}
 
+{% block body %}
 
+	{% if "now"|date("Y/m/d") >= decisionFirstPeriod|date("Y/m/d")  %}
+		{% set backgroundColor = 'bg-tertiary' %}
+	{% elseif "now"|date("Y/m/d") >= decisionConflictPeriod|date("Y/m/d")  %}
+		{% set backgroundColor = 'bg-cinco' %}
+	{% elseif "now"|date("Y/m/d") >= decisionFinalPeriod|date("Y/m/d")  %}
+		{% set backgroundColor = 'bg-primary' %}
+	{% endif %}
 	<main>
-		<div class="container py-5 rounded-4 mt-5 mb-5 bg-quatro">
-			<div class="row py-3">
-				<h1 class="text-secondary text-uppercase fw-bolder fs-4 col-md-8 text-center">{{ decision.title }}</h1>
-
-				<div class="col-md-4 text-white d-flex pt-4">
-					<img class="mx-2 pictureCreator" src="{{ asset('build/images/icons/person.svg') }}" alt="Image de l'auteur">
-					<div class="ms-2 py-3 pe-2">par
-						{{decision.creator.firstname ~ ' ' ~ decision.creator.lastname}}</div>
-					<div class="text-uppercase fw-bold py-3"></div>
+		<div class="mt-5 container text-center">
+			<h1 class="fs-1 fw-bold py-3">Avis</h1>
+		</div>
+
+
+		{% if "now"|date("Y/m/d") >= decisionFirstPeriod|date("Y/m/d")  %}
+			<div class="container-fluid">
+				<div class="bg-tertiary py-4 rounded-top">
+					<div class="border-bottom border-white border-3 mb-3 mx-5">
+						<h2 class="text-center text-white mt-3">Début de la période pour donner votre avis</h2>
+					</div>
 				</div>
-			</div>
+			{% endif %}
+
+			{% for commentsFirstPeriod in commentsFirstPeriods %}
```

Comment:
fait un include qui contient toute cette boucle for, en passwant dans le "with" comments et bg-color
ça simplifiera au moins un peu